maxim/logger/livekit/utils.py

```
@@ -1,4 +1,8 @@
+import copy
```

Ruff (0.11.9)

`copy` imported but unused

Remove unused import: `copy`

(F401)